test/ponylang-mode-test.el

```diff
@@ -1,29 +1,4 @@
-;;; ponylang-test.el --- Tests for ponylang-mode
-;;
-;; Author: Sean T Allen <sean@monkeysnatchbanana.com>
-;; Version: 0.6.0
-;; URL: https://github.com/seantallen/ponylang-mode
-;; Keywords: programming
-;; Package-Requires: ()
-;;
-;; This file is not part of GNU Emacs.
-;;
-;; Copyright (c) 2015 Austin Bingham
-;; Copyright (c) 2016 Sean T. Allen
-;; Copyright (c) 2020 Damon Kwok
-;; Copyright (c) 2021 The Pony Developers
-;;
-;;; Commentary:
-;;
-;; Description:
-;;
-;; Tests for ponylang-mode
-;;
-;; The tests are built using the standard `ert' module, so you can run
-;; them using a number of techniques. For example:
-;;
-;;     (require 'ponylang-test)
-;;     (ert-run-tests-interactively "ponylang-test")
```

Comment:
why the removal of all this?

Reply:
As a test file, it is not supposed to be shipped with the package and because it has some properties like a real package, Eldev thinks it is another package in the same repository. 

I will re-add the commentary and update it to include eldev instructions it you think it is pertinent but I think we shouldn't include these package attributes, eg: `Author:`, ` Version:`, etc.